src/Spout/Reader/XLSX/Helper/SharedStringsHelper.php

```diff
@@ -60,15 +60,28 @@ public function __construct($filePath, $tempFolder = null)
      */
     public function hasSharedStrings()
     {
-        $hasSharedStrings = false;
+        return $this->getSharedStringsPath() !== null;
+    }
+
+    /**
+     * Returns shared strings XML file path
+     *
+     * @return string|null
+     */
+    public function getSharedStringsPath()
+    {
+        $sharedStringsPath = null;
         $zip = new \ZipArchive();
 
         if ($zip->open($this->filePath) === true) {
-            $hasSharedStrings = ($zip->locateName(self::SHARED_STRINGS_XML_FILE_PATH) !== false);
+            $index = $zip->locateName(self::SHARED_STRINGS_XML_FILE_PATH, \ZipArchive::FL_NOCASE);
```

Comment:
instead of using a constant and try to find the correct file name by locating it inside the zip file, we should probably get it from the `[Content_Types].xml` file.

It looks something like this:
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Types xmlns="http://schemas.openxmlformats.org/package/2006/content-types">
    ....
    <Override PartName="/xl/sharedStrings.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.sharedStrings+xml"/>
</Types>
```